update/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# Input variables
+LOCAL_NCBI_DIR=$(pwd)/ncbi
+PREVIOUS_UTA_VERSION=uta_20210129b
+
+# 1. Download files for SeqRepo and UTA updates
+# Input: N/A
+# Output: LOCAL_NCBI_DIR populated with downloaded files
+mkdir -p $LOCAL_NCBI_DIR
+docker build --file Dockerfile.download --tag ncbi:latest --progress plain .
+docker run -it --mount type=bind,source=$LOCAL_NCBI_DIR,target=/data/ncbi ncbi:latest
+
+# 2. Prepare SeqRepo
+# Input: TBD
```

Comment:
This could be...
```
SEQREPO_DATA_RELEASE='2024-2-20'

docker pull biocommons/seqrepo:$SEQREPO_DATA_RELEASE
docker run --name seqrepo biocommons/seqrepo:$SEQREPO_DATA_RELEASE

Reply:
Done in #9 in `sbin/seqrepo-download`